### DISCUSSION TITLE
OpenFGA upgrade from 0.2.4 to 0.3.7. SDK-Generator openapi docs version?

### DISCUSSION THREAD

**Discussion by uzubair-miovision** [openfga/community#204] [Q&A]
Hello all, I'm in the process of updating OpenFGA from version 0.2.4 to 0.3.7. As part of this upgrade, I need to create Kotlin clients using the repository found at https://github.com/openfga/sdk-generator. However, I'm facing an issue regarding the appropriate tag for the `OPEN_API_URL`, which is currently set to `https://raw.githubusercontent.com/openfga/api/main/docs/openapiv2/apidocs.swagger.json`. I'm unsure if the latest version is compatible with my target version, 0.3.7. If it isn't, could someone please advise me on which specific SHA to utilize? Any assistance on this matter would be greatly appreciated. Thank you.

**Comment by rhamzeh** [comment]
Hi @uzubair-miovision - Can you let us know what issue you are facing? Is the generator not working?
